### ISSUE TITLE
 Conversational Actions

### ISSUE THREAD

**Issue by jmndoza** [armilam/google-assistant-ruby#36]
Hi!   it seems like you use the App Actions and not [Conversational](https://developers.google.com/assistant/conversational/prompts#format_of_a_prompt) ones, right?
i been struggle with different JSON request 😔

**Comment by armilam** [comment]
You're right! This is a very old and out of date project. Feel free to help
me update it with some PRs if you like, though.
